packages/workflows/src/ios.workflows.ts

```diff
@@ -19,6 +19,20 @@ const childWorkflows: IDeploymentWorkflows = {
       },
     ],
   },
+  set_icons_and_splash_images: {
+    label:
+      "Generate app launcher icon and splash screen image from icon.png and splash.png asset respectively. Then if provided, it will generate adaptive icon from icon-foreground.png and icon-background.png. Copy generated files to relevant folders.",
```

Comment:
I think again that this should be "icon-only.png" instead of "icon.png". And no harm in linking to the docs again... Also copy over this comment to corresponding Android equivalent

Reply:
I think this one may have to be changed within the gdrive. The comment I have changed but the actual name itself might have to be changed.